README.md

```diff
@@ -78,6 +78,10 @@ Below you will find descriptions of all rules available in **AEM Rules for Sonar
 - **AEM-19** Implicit search strategy used in Sling Query
   - `SearchStrategy` can have negative performance impact if mismatched.
   Therefore developer should always make informed decision and define strategy explicitly.
+  
+- **HTL-3** Use Explicit Names in Loops
```

Comment:
Shouldn't be done like here https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/137/files#diff-04c6e90faac2675aa89e2176d2eec7d8 ?

Second question. I see duplicated id of rule:
https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/124/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Please fix were needed

Reply:
done, but should be discussed.